src/SqlSquared/Signature.purs

```diff
@@ -138,8 +142,20 @@ data SqlF literal a
   | Select (SelectR a)
   | Parens a
 
+type AnyDirPath = E.Either (Pt.AbsDir Pt.Unsandboxed) (Pt.RelDir Pt.Unsandboxed)
+
+printAnyDirPath :: AnyDirPath -> String
+printAnyDirPath = E.either Pt.unsafePrintPath Pt.unsafePrintPath
+
+parseAnyDirPath :: forall m. Applicative m => (forall a. String -> m a) -> String -> m AnyDirPath
+parseAnyDirPath fail = Pt.parsePath
+  (pure ∘ E.Right)
+  (pure ∘ E.Left)
+  (const $ fail "incorrect directory path")
+  (const $ fail "incorrect directory path")
```

Comment:
Nitpick: I think "incorrect directory path" is an easy error for users to misinterpret. Perhaps "Expected a directory path"?

Reply:
Absolutely!